app/views/pages/get-started.liquid

```diff
@@ -38,13 +38,64 @@ Besides standard web technologies like HTML, CSS, JavaScript, platformOS uses a
 - [Get started with YAML in 5 minutes](https://www.educative.io/blog/yaml-tutorial)
 
 
-
-## Hello, World!
-
-[Hello, World!](/get-started/hello-world/hello-world) is the first part of our Get Started guide, that will teach you how to set up your development environment, create a simple site, make a small change on the home page, and deploy it. This is to make sure you have everything you need and everything works. You can stop here and start experimenting based on our documentation, or you can move on to the second part of our Get Started guides.
-
-## Build a ToDo List App
-
-[Build a ToDo List App](/get-started/todo-app/build-todo-list-app) is the second part of our Get Started guide that walks you through the steps of creating a ToDo List app on platformOS from setting up your development environment to deploying and testing your finished app. It explains basic concepts, main building blocks, and the logic behind platformOS, while also giving you some recommendations on workflow.
-
-[Build a ToDo List App - Part 2: Extend Your App](/get-started/todo-app/build-todo-list-app-part-2) shows you how to extend what you have built in part 1 to explore additional platformOS features in greater depth and learn about more best practices.
+## Not sure where to start?
+
+Let’s build a simple website together! If you are new to platformOS, we propose that you start with a simple course that we’ve prepared.
+
+You will learn how to set up your environment, how to get a starter codebase and how to deploy everything to your platfromOS instance. Next, we’ll gona build a simple to-do app using all of the core platformOS concepts.
+
+Ready? Let’s learn how to [install and configure](/get-started/installation-and-configuration/) your development environment.
+
+
+## Quick start
+
+1. Go to Partner Portal and [sign up](https://partners.platformos.com/accounts/sign_up)
+2. [Create new instance](https://partners.platformos.com/instances/new) in Partner Portal
+3. Install pos-cli
+    platformOS command-line interface is distributed through NPM.
+  <pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
+npm install -g @platformos/pos-cli
+  </code></pre>
+4. Create an empty folder for your project
```

Comment:
Thought about it, but since we are assuming someone might have just a basic cmd knowledge, I assumed it would be easier just to make people create a folder from the UI and not having to describe what `mkdir` does as this is not the scope of the tutorial. I am describing every other command that's being used so I would leave it for the users.